WMF Framework/UILabel+HasText.swift

```diff
@@ -0,0 +1,5 @@
+extension UILabel {
+    public var wmf_hasText: Bool {
+        return (text as NSString?)?.length ?? 0 > 0
```

Comment:
Was NSString used here because of issues related to Swift strings not currently being collections of characters and how that makes character counts screwy in some langs?